reporting/miro_transformer/src/transform.py

```diff
@@ -26,7 +26,7 @@ def clean_dates(data, keys_to_parse):
     alphabetically is rubbish
     """
     for key, value in data.items():
-        if value_should_be_parsed(key, value, keys_to_parse):
+        if any(key == substring for substring in keys_to_parse):
```

Comment:
I think the following is equivalent, and simpler:
```suggestion
        if key in keys_to_parse:
```